### PR TITLE
Yoan's branch

### PR DESCRIPTION
**Pull request by HackerstoyanovBG**

### CODE REVIEW

**Review by MitkoVtori — CHANGES_REQUESTED** [review]
You miss spelled somethings #games

**Review by MitkoVtori — CHANGES_REQUESTED** [review]
You miss spelled the docs games also

File: (E) Progress/progress.html [hunk 1]
```diff
@@ -11,9 +11,7 @@ <h3 style="margin: 0; background-color: rgb(0, 0, 0);">
             <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(27, 164, 255);">Certificates</a>
             <a href="./progress" style="margin-left: 1%; font-size: 30px; color: rgb(27, 164, 255);">Progress</a>
             <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">School</a>
-            <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 35%; font-size: 50px;">contact</a>
-                <div class="expandable" id="navigation">
-                    <a href="./games" style="color: rgb(241, 255, 51);">Games page</a>
+            <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a><a href="https://mitkovtori-github-io.translate.goog/Mitko.Vtori-World/?_x_tr_sl=en&_x_tr_tl=bg&_x_tr_hl=bg&_x_tr_pto=wapp"><button style="font-size: 40px; margin-left: 4%;">Translate This Page</button></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 7%; font-size: 50px;">contact</a>
```
Comment (MitkoVtori) [inline (E) Progress/progress.html:14]: Alright

Comment (MitkoVtori) [reply (E) Progress/progress.html:14]: .... I think this is okay

File: (C) games/(1) games.html [hunk 2]
```diff
@@ -24,8 +23,7 @@ <h3 style="margin: 0; background-color: rgb(50, 50, 248);">
         <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(251, 255, 27);">Certificates</a>
         <a href="./progress" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">Progress</a>
         <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">School</a>
-        <a href="./games" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">Games</a>
```
Comment (MitkoVtori) [inline (C) games/(1) games.html:27]: This is a little bit not okay but I can live with it

File: docs/games.html [hunk 3]
```diff
@@ -23,7 +22,7 @@ <h3 style="margin: 0; background-color: rgb(50, 50, 248);">
         <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(251, 255, 27);">Certificates</a>
         <a href="./progress" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">Progress</a>
         <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">School</a>
-        <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a><a href="https://mitkovtori-github-io.translate.goog/Mitko.Vtori-World/?_x_tr_sl=en&_x_tr_tl=bg&_x_tr_hl=bg&_x_tr_pto=wapp"><button style="font-size: 40px; margin-left: 4%;">Translate This Page</button></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 7%; font-size: 50px;">contact</a>
```
Comment (MitkoVtori) [inline docs/games.html:26]: Fixed the issue

Comment (MitkoVtori) [reply docs/games.html:26]: Also renamed

File: (B) certificates/certificates.html [hunk 4]
```diff
@@ -11,7 +11,7 @@ <h3 style="margin: 0; background-color: rgb(50, 50, 248);">
             <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(251, 255, 27);">Certificates</a>
             <a href="./progress" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">Progress</a>
             <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">School</a>
-            <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 35%; font-size: 50px;">contact</a>
+            <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a><a href="https://mitkovtori-github-io.translate.goog/Mitko.Vtori-World/?_x_tr_sl=en&_x_tr_tl=bg&_x_tr_hl=bg&_x_tr_pto=wapp"><button style="font-size: 40px; margin-left: 4%;">Translate This Page</button></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 7%; font-size: 50px;">contact</a>
```
Comment (MitkoVtori) [inline (B) certificates/certificates.html:14]: That's a little bit too long, but okay

**Review by MitkoVtori — APPROVED** [review]
Okay, everything is clean 👍

**Comment by HackerstoyanovBG** [comment]
May I pull the request to the main branch?

File: (D) Home/Home.html [hunk 5]
```diff
@@ -11,7 +11,7 @@ <h3>
                 <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(11, 255, 255);">Certificates</a>
                 <a href="progress" style="margin-left: 1%; font-size: 30px; color: rgb(11, 255, 255);">Progress</a>
                 <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(11, 255, 255);">School</a>
-                <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(11, 255, 255);"><span>Games</span></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 35%; font-size: 50px;">contact</a>
```
Comment (MitkoVtori) [inline (D) Home/Home.html:14]: Okay that's clean...

Comment (MitkoVtori) [reply (D) Home/Home.html:14]: also still looks too long

**Review by MitkoVtori — APPROVED** [review]
Alright

**Pull request merged**